docs/ref/django-admin.txt

```diff
@@ -677,6 +677,11 @@ is:
 
 Requires ``gettext`` 0.19 or newer.
 
+.. django-admin-option:: --no-obsolete
+
+Removes all obsolete message strings from the resulting ``.po`` files when 
+atleast one string is marked for translation. 
```

Comment:
I'd simplify it:
```suggestion
Removes obsolete message strings from the ``.po`` files.
```
_" when atleast one string is marked for translation"_  can be misleading for users and is not precise as we need at least one translatable string per file. I'd remove it.